config/35-pilot.ini

```diff
@@ -0,0 +1,39 @@
+;===================================================================
+;                          Pilots
+;===================================================================
+
+; For each pilot type, add a new pilot section.
+; Each pilot name must be unique for the entire grid, so make sure to not
+; pick anything generic like "MAIN".
```

Comment:
Do you know what the pilot name corresponds to (e.g. a value in the factory)? What happens if that name /isn't/ unique? How does a site determine what names are available?

Reply:
Since this is going into OSG_ResourceCatalog, I guess you can query the CE collector; I don't know what naming scheme the factory folks will use.

One idea might be to prepend the topology resource name (from 40-siteinfo.ini) to the name, which gets rid of the global uniqueness requirement...

Reply:
Could you drop an example of what the name looks like in the resource catalog? That way if @mmascher is using it for entry generation, we determine what this name corresponds to in the factory/accounting/etc

Reply:
It's put as-is into an attribute named `Name` in each resource catalog entry -- see the example in this comment: https://opensciencegrid.atlassian.net/browse/SOFTWARE-4177?focusedCommentId=360927

Reply:
I tweaked the comment.